catboost/pytest/test.py

```diff
@@ -7011,7 +7011,7 @@ def run_catboost(eval_path, mvs_head_fraction):
             '--has-header',
             '--boosting-type', boosting_type,
             '--bootstrap-type', 'MVS',
-            '--mvs-head-fraction', mvs_head_fraction,
+            '--subsample', mvs_sample_rate,
```

Comment:
Could you add mvs parameters to python-package/catboost/core.py to CatBoostClassifier and CatBoostRegressor __init__ function?